### PR TITLE
concatenate_recordings / append_recordings

### PR DESCRIPTION
**Pull request by samuelgarcia**
@alejoe91 : tell me what do you think.

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Hi Alessio. Could you merge this ?

File: spikeinterface/core/segmentutils.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+import numpy as np
+
+from .baserecording import BaseRecording, BaseRecordingSegment
+from .basesorting import BaseSorting, BaseSortingSegment
+from .baseevent import BaseEvent, BaseEventSegment
+
+
+class AppendSegmentRecording(BaseRecording):
+    """
+    Return a recording that "append" all segments from all recording
```
Comment (alejoe91) [inline spikeinterface/core/segmentutils.py:21]:
```suggestion
    Return a recording that "appends" all segments from all recording
```

File: spikeinterface/core/segmentutils.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        return self.parent_segment.get_traces(*args, **kwargs)
+
+def append_recordings(*args, **kargs):
+    return AppendSegmentRecording(*args, **kargs)
+append_recordings.__doc__ == AppendSegmentRecording.__doc__
+
+
+class ConcatenateSegmentRecording(BaseRecording):
+    """
+    Return a recording that "concatenate" all segments from all recording
```
Comment (alejoe91) [inline spikeinterface/core/segmentutils.py:75]:
```suggestion
    Return a recording that "concatenates" all segments from all recording
```

File: spikeinterface/core/segmentutils.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    into one recording mono segment. Operation is lazy.
```
Comment (alejoe91) [inline spikeinterface/core/segmentutils.py:76]:
```suggestion
    into one recording mono segment. The operation is lazy.
```

**Review by alejoe91 — APPROVED** [review]
@samuelgarcia just have a few comments with typos docs improvements!

**Pull request merged**